bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html

```diff
@@ -183,11 +183,11 @@ <h4 class="section-title" id="drumbeat">
       <p>
         {% trans
           logo=static('img/foundation/annualreport/2009/drumbeat-logo.jpg'),
-          webcraft="http://www.drumbeat.org/",
-          festival="http://www.drumbeat.org/",
-          movies="http://www.drumbeat.org/",
-          subtitles="http://www.drumbeat.org/",
-          fund="https://donate.mozilla.org/"
+          webcraft="//www.drumbeat.org/webcraft",
```

Comment:
Bah! Yes, there was a merge conflict and I wanted to keep my change to use the `donate_url` and did not notice I copied over that one as well. Fixing. Apologies for that @alexgibson